code/modules/antagonists/ninja/ninja.dm

```diff
@@ -105,7 +105,7 @@
 	return TRUE
 
 /proc/is_ninja(mob/living/M)
-	return M && M.mind && M.mind.has_antag_datum(/datum/antagonist/ninja)
+	return M?.mind?.has_antag_datum(/datum/antagonist/ninja)
```

Comment:
Are you sure this actually fixes the empty objective bug? `M && M.mind && M.mind.has_antag_datum(/datum/antagonist/ninja)` and `M?.mind?.has_antag_datum(/datum/antagonist/ninja)` are just two different ways of writing the same code. Unless BYOND is being stupid and returning `M` for some reason.

Reply:
Oh nevermind I'm a smoothbrain. The quantity was the objective issue.